packages/e2e-cypress/src/templates/typescript/cypress.config.ts

```diff
@@ -27,5 +27,7 @@ export default defineConfig({
     specPattern: 'src/**/*.cy.{js,jsx,ts,tsx}',
     indexHtmlFile: 'test/cypress/support/component-index.html',
     devServer: injectQuasarDevServerConfig(),
+    <% if (requiresPublicPath) {%>// @ts-expect-error -- If not set it will break tests related to components that load public assets. See https://github.com/quasarframework/quasar-testing/issues/379
+    devServerPublicPathRoute: '' <% } %>

```

Comment:
@IlCallo in the future, it might be a good idea to expose a function to not only the dev server, but an object with the dev server being one property. That way, it would be possible to abstract away some details like this. But, for now, this should be good.

Reply:
Ok, please add this as an issue on this repo and leave a reference to it as a TODO in the code

We'll tackle it as soon as we start working on a new major release